src/EventTypesView/EventQueryListItem.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import IconButton from 'material-ui/IconButton';
+import IconEdit from 'material-ui/svg-icons/editor/mode-edit';
+import IconDelete from 'material-ui/svg-icons/action/delete';
+import {ListItem} from 'material-ui/List';
+
+
+class EventQueryListItem extends Component {
+	deleteEventQuery() {
+		// TODO: to be filled
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1135)